pkg/controller/volume/cache/actual_state_of_world.go

```diff
@@ -47,7 +55,7 @@ type ActualStateOfWorld interface {
 	// added.
 	// If no node with the name nodeName exists in list of attached nodes for
 	// the specified volume, the node is added.
-	AddVolumeNode(volumeSpec *volume.Spec, nodeName string) (api.UniqueDeviceName, error)
+	AddVolumeNode(volumeSpec *volume.Spec, nodeName string) (api.UniqueVolumeName, error)
```

Comment:
Rename to `AddVolumeToNode` and check all interfaces like this for naming problems